### DISCUSSION TITLE
Question about the DB or something else

### DISCUSSION THREAD

**Discussion by GreenSage** [dbeaver/dbeaver#13245] [Q&A]
Hello everyone. It's been a long time since I've posted an issue/chat/QA because the software has been awesome and no issues. :) But I was hoping if I may borrow a few seconds of your time for a question related to MariaDB or DBeaver I'm not sure which. I realize it may not be specifically DBeaver related but as I've moved from shared hosting to cloud hosting (and thus having to manage all the knobs and levers) something happened when I got this new shiney laptop from wally world lol. I installed DBeaver and my 3 databases and all the tables and everything is great no issues. But I use an open source server monitoring system and it shows me this now:

information_schema | mysql |   | performance_schema |   | [db's listings | 

Should I be concerned because never before was "mysql" listed only the 2 schemas and my own databases. I've watched this github for a while and everyone seems very knowledgeable maybe able to help outside the scope on this one? I'm extremely anxious and worried. I've used DBeaver for just over 2 years now and this "mysql" has never showed up!? I've looked into it and it's far far too technical for me so I won't touch any of it.

**Comment by LonwoLonwo** [comment]
Hello @GreenSage 
As I can see, "mysql" is a system schema: https://dev.mysql.com/doc/refman/8.0/en/system-schema.html
You can check it with this statement: "show databases"

So if you think DBeaver created it - then no, it was not us :)
